api/db/repositories/orderHelpers/orderHelpers.js

```diff
@@ -0,0 +1,99 @@
+const orderDomain = require('@domains/ordersDomain')
+const orderRepo = require('@repos/orderRepository')
+const productRepo = require('@repos/productRepository')
+const { Verification, Resources } = require('@verify/verification')
+
+class OrderHelpers {
```

Comment:
names of functions in this class need to be changed because they are totally not describing what they do. Like AddCounter could be `updateQuantityAndTotalPrice` etc...